pkg/ccl/backupccl/alter_backup_schedule_test.go

```diff
@@ -139,4 +140,18 @@ INSERT INTO t1 values (1), (10), (100);
 	},
 		backupStmts)
 
+	rows = th.sqlDB.Query(t,
+		fmt.Sprintf(`ALTER BACKUP SCHEDULE %d EXECUTE FULL IMMEDIATELY;`, scheduleID))
+
+	require.NoError(t, rows.Err())
+	require.True(t, rows.Next())
+	require.NoError(t, rows.Scan(&scheduleID, &unusedStr, &status, &nextRun, &schedule, &backupStmt))
+
+	require.True(t, rows.Next())
+	require.NoError(t, rows.Scan(&scheduleID, &unusedStr, &status, &nextRun, &schedule, &backupStmt))
+
+	require.True(t, nextRun != nil)
+	t.Log(nextRun)
+	require.True(t, th.env.Now().Equal(*nextRun))
```

Comment:
Can we use `require.Equal(t, th.env.Now(), *nextRun)` and then remove the log line since the expectation output will have the expected time and actual time?

Reply:
We can't: time.Time structs are not comparable other than via their methods due to monotonic/locale/etc

Reply:
For some reason I thought `require` had some code to call Equal if it exists, but it looks like it just uses reflect.DeepEqual. Bummer.